dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -43,10 +43,15 @@ class ScriptsControllerTest < ActionController::TestCase
   end
 
   test "should get show of k-8" do
-    get :show, id: Script::TWENTY_HOUR_ID
+    get :show, id: '20-hour'
```

Comment:
Good catch, this constant already exists in script.rb.  I'll update here to use it.